static/app/api.tsx

```diff
@@ -502,6 +506,7 @@ export class Client {
           responseJSON,
           responseText,
           getResponseHeader: (header: string) => response.headers.get(header),
+          rawResponse: response.clone(),
```

Comment:
We probably have to clone this earlier in the code?

Reply:
Can we try replacing the `response.text()` call with `response.clone().text()` so that we never try to fetch the body of `response` directly?